### PR TITLE
Update transcoder events monitoring / stats reporting

### PR DESCRIPTION
**Pull request by ericxtang**
Make sure the CLI is showing the right information

### CODE REVIEW

File: server/webserver.go [hunk 1]
```diff
@@ -703,7 +703,17 @@ func (s *LivepeerServer) StartWebserver() {
 		if s.LivepeerNode.Eth != nil && s.LivepeerNode.EthEventMonitor != nil {
 			activeEventSubMap := s.LivepeerNode.EthEventMonitor.EventSubscriptions()
```
Comment (yondonfu) [inline server/webserver.go]: Should we check if the JobService is working in the same way that we are checking that the RewardService and RoundService are working?

**Review by yondonfu — COMMENTED** [review]
Left 1 comment

**Review by yondonfu — APPROVED** [review]
LGTM 🚢

File: cmd/livepeer_cli/wizard_eth.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+)
+
+func (w *wizard) setGasPrice() {
```
Comment (yondonfu) [inline cmd/livepeer_cli/wizard_eth.go:8]: Wasn't this included in a previous merged PR?

Comment (ericxtang) [reply cmd/livepeer_cli/wizard_eth.go:8]: I thought so too... git confuses me sometimes.

**Pull request merged**